Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocList.css

```diff
@@ -386,8 +386,8 @@ div.bocList .bocListCellStructureElement > .validationErrorMarker
   padding: calc(var(--remotion-themed-spacing) / 2);
 }
 
-div.bocList .bocListCellStructureElement > .validationErrorMarker:focus,
-div.bocList .bocListContent > .validationErrorMarker:focus
+div.bocList .bocListCellStructureElement > .validationErrorMarker:focus-within,
```

Comment:
NovaGray also has `.bocList .validationErrorMarker:focus`. I believe that needs updating, too?